samples/Meeting/src/app/utils/getEndpointUrl.ts

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+let endpointUrl: string | undefined;
```

Comment:
(presumably legacy code from chat) but very odd to have a global variable for this, can you safely remove this?

Reply:
Oh its used for caching this value... we should still remove the global variable and it can be cached in a useState if its being retrieved frequently